tools/wasme/cli/example/assemblyscript/runtime-config.json

```diff
@@ -2,7 +2,8 @@
   "type": "envoy_proxy",
   "abiVersions": [
     "v0-541b2c1155fffb15ccde92b8324f3e38f7339ba6",
-    "v0-097b7f2e4cc1fb490cc1943d0d633655ac3c522f"
+    "v0-097b7f2e4cc1fb490cc1943d0d633655ac3c522f",
+    "v0.2.1"
```

Comment:
not sure if asm scripts bundled here supports this version; either update asm script here, or revert this version change?